catalog/ui/catalog-ui-search/src/main/webapp/component/workspace-item/workspace-item.collection.view.js

```diff
@@ -53,15 +53,15 @@ define([
             switch (getPrefs().get('homeSort')) {
                 case 'Title':
                     return workspace.get('title').toLowerCase();
-                case 'Last modified':
                 default:
-                    return workspace.get('modified');
+                    return -workspace.get('metacard.modified');
```

Comment:
what does `-` do here?

Reply:
Pretty sure it switches the sorting order in a funny "switch the sign of a value going into a comparator" kind of way.